### PR TITLE
Han Temple

### PR DESCRIPTION
**Pull request by indoptogopt**

### CODE REVIEW

File: 0-ad-history-mod-package/simulation/templates/structures/han/temple.xml [hunk 1]
```diff
@@ -10,6 +10,13 @@
   <Identity>
     <Civ>han</Civ>
     <SpecificName>Sìmiào</SpecificName>
+    <History>
+        The Han Empire's vast lands were places of cultural diversity and home to many beliefs and religions. The most widespread ones were Taoism and Buddhism. However, the central imperial government followed a legalistic and confucianistic ideology. Legalism claims the necessity of law, justice, and control, while Confucianism is all about morale, about reigning gently and obeying resistancelessly:
+        The Han political system was very complex but clearly structured. There was a very strict hierarchy, and great respect was demanded for higher-ranking people. Han emperors were, unlike many other emperors in history, not worshipped as gods but seen as Mandates of Heaven representing the gods.
+        But not only the official administration held a strong hierarchy; following the Confucian model, the man would lead a family, women and kids had little freedom, were clearly subordinate to the man, and had to obey him.
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/structures/han/temple.xml:16]: OK, this paragraph I have problems with. But I'll discuss more in the forums, but even so the wording is really negative.

```suggestion
        It wasn't only the official administration that held a strong hierarchy. Following the Confucian model, the man would be the family's leader who women and children had to obey.
```

File: 0-ad-history-mod-package/simulation/templates/structures/han/temple.xml [hunk 2]
```diff
@@ -10,6 +10,13 @@
...
+        The highest goal of this system, based on Confucian teachings, was to establish order, harmony was believed to be the foundation for an ideal society.
+        And it worked. At least for a certain time. Especially the early days of the Han Empire were times of peace (at least within the Empire's borders) and flourishing of Chinese culture. This time was certainly not without reason called “The Golden Age of China”.
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/structures/han/temple.xml:18]:
```suggestion
```

**Review by TheShadowOfHassen — COMMENTED** [review]
Content Edit:

Pretty good except for one problem. I will post the reasons on the forums.

**Comment by TheShadowOfHassen** [comment]
Also real quick when looking over. Did the Han have any sacrifices in either of their religions? Or anything else that impacted their culture?

**Comment by TheShadowOfHassen** [comment]
Yes a mention of that would be good.

**Comment by indoptogopt** [comment]
I replace that one paragraph. Is it better like this?

File: 0-ad-history-mod-package/simulation/templates/structures/han/temple.xml [hunk 3]
```diff
@@ -10,6 +10,12 @@
   <Identity>
     <Civ>han</Civ>
     <SpecificName>Sìmiào</SpecificName>
+    <History>
+        The Han Empire's vast lands was a place of cultural diversity, and home to many beliefs and religions. The most widespread ones were Taoism and Buddhism. However, the central imperial government followed a legalistic and confucianistic ideology. Legalism claims the necessity of law, justice, and control, while Confucianism is all about morality, reigning gently and obeying.
+        The Han political system was complex but clearly structured. There was a very strict hierarchy, and great respect was demanded for higher-ranking people. Han emperors were, unlike many other emperors in history, not worshipped as gods but seen as Mandates of Heaven, representing the gods on earth.
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/structures/han/temple.xml:15]:
```suggestion
        The Han political system was complex but clearly structured. There was a very strict hierarchy, and great respect was demanded for higher-ranking people. Han emperors were, unlike many other emperors in history, not worshiped as gods but seen as Mandates of Heaven, representing the gods on earth.
```
You used the British spelling for worship

File: 0-ad-history-mod-package/simulation/templates/structures/han/temple.xml [hunk 4]
```diff
@@ -10,6 +10,12 @@
...
+        Deceased emperors were often buried in extravagant tombs, like giant mausoleums. But also among common people, ancestor worship was also widely practiced. It was customary for families to bring sacrifices or perform special rituals in honor of their ancestors. 
+        The highest goal of this system, based on Confucian teachings, was to establish order, harmony was believed to be the foundation for an ideal society.  And it worked. At least for a certain time. Especially the early days of the Han Empire were times of peace (at least within the Empire's borders) and flourishing of Chinese culture. This time was certainly not without reason called “The Golden Age of China”.
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/structures/han/temple.xml:17]:
```suggestion
        The highest goal of this system, based on Confucian teachings, was to establish order, harmony was believed to be the foundation for an ideal society.  And it worked. At least for a certain time. Especially during the early days, the Han Empire was a place of peace (at least within the Empire's borders) and in it Chinease culture flourished. This time was certainly not without reason, called “The Golden Age of China”.
```

**Review by TheShadowOfHassen — COMMENTED** [review]
Pretty good, this is the copyedit. When you're done making these changes it can be merged.

File: 0-ad-history-mod-package/simulation/templates/structures/han/temple.xml (hunk 3 above)
Comment (indoptogopt) [reply 0-ad-history-mod-package/simulation/templates/structures/han/temple.xml:15]: Yeah, my bad

**Pull request merged**